frontend/src/component/project/Project/ProjectInfo/ToggleTypesWidget.tsx

```diff
@@ -0,0 +1,81 @@
+import { IFeatureToggleListItem } from '../../../../interfaces/featureToggle';
+import { useMemo } from 'react';
+import {
+    StyledCount,
+    StyledDivInfoContainer,
+    StyledParagraphGridRow,
+    StyledParagraphSubtitle,
+} from './ProjectInfo.styles';
+import { getFeatureTypeIcons } from '../../../../utils/getFeatureTypeIcons';
+
+export interface IToggleTypesWidgetProps {
+    features: IFeatureToggleListItem[];
+}
+
+export const ToggleTypesWidget = ({ features }: IToggleTypesWidgetProps) => {
+    const { release, experiment, operational, kill, permission } =
+        useMemo(() => {
+            const release =
+                features?.filter(feature => feature.type === 'release')
```

Comment:
can features be empty? the type doesn't indicate that

Reply:
It can, if it is a new project for example

Reply:
but then it's empty array not undefined. So maybe the question mark is not needed.